composer.json

```diff
@@ -25,5 +25,10 @@
         "psr-4": {
             "GiveDivi\\": "src/"
         }
+    },
+    "config": {
+        "platform": {
+            "php": "5.6.40"
+        }
```

Comment:
@alaca I update it to make `composer install` independent on the system environment.